### PR TITLE
Remove redundant parentheses

### PR DESCRIPTION
**Pull request by m14815**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
some parentheses are redundant, and should not be useful
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: cmd/kube-apiserver/app/options/validation.go [hunk 1]
```diff
@@ -58,7 +58,7 @@ func validateClusterIPFlags(options *ServerRunOptions) []error {
 	// while api-server dualstack bits does not have dependency on EndPointSlice, its
 	// a good idea to have validation consistent across all components (ControllerManager
 	// needs EndPointSlice + DualStack feature flags).
-	secondaryServiceClusterIPRangeUsed := (options.SecondaryServiceClusterIPRange.IP != nil)
+	secondaryServiceClusterIPRangeUsed := options.SecondaryServiceClusterIPRange.IP != nil
```
Comment (lala123912) [inline cmd/kube-apiserver/app/options/validation.go:61]: the original one will be more readable

Comment (cici37) [reply cmd/kube-apiserver/app/options/validation.go:61]: Agree with Melisa

**Pull request closed without merging**